### DISCUSSION TITLE
Welcome to Edwood Discussions!

### DISCUSSION THREAD

**Discussion by rjkroege** [rjkroege/edwood#322] [General]
## 👋 Hello!
For a while, I've thought that it would be nice to have somewhere to discuss Edwood, its development, features etc. Conveniently GitHub introduced a discussions feature so seems like worth giving it a try.

**Comment by rjkroege** [comment]
I should figure out how to make the background of the pinned post the standard Acme/Edwood "white".

**Comment by paul-lalonde** [comment]
What a great idea!  A little e friendlier than filling bugs
